datasets/billsum/README.md

```diff
@@ -132,6 +132,8 @@ The data fields are the same among all splits.
 
 ### Source Data
 
+The data consists of three parts: US training bills, US test bills and California test bills. The US bills were collected from the [Govinfo](https://github.com/unitedstates/congress) service provided by the United States Government Publishing Office (GPO) under CC0-1.0 license. The California, bills from the 2015-2016 session were are from the legislature’s [website](https://leginfo.legislature.ca.gov/).
+
 #### Initial Data Collection and Normalization
 
 [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
```

Comment:
This information should be placed under the subsection "Initial Data Collection and Normalization":
```suggestion
#### Initial Data Collection and Normalization

The data consists of three parts: US training bills, US test bills and California test bills. The US bills were collected from the [Govinfo](https://github.com/unitedstates/congress) service provided by the United States Government Publishing Office (GPO) under CC0-1.0 license. The California, bills from the 2015-2016 session were are from the legislature’s [website](https://leginfo.legislature.ca.gov/).
```